migrations/20220217180600-drop-photos-tables.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  up: async (queryInterface) => {
+    await queryInterface.dropTable('previews');
+    await queryInterface.dropTable('photosalbums');
+    await queryInterface.dropTable('albums');
+    await queryInterface.dropTable('photos');
+    await queryInterface.dropTable('usersphotos');
+  },
+
+  down: async () => {
+    // Nothing to do
```

Comment:
Here the reverse behavior for the `up` method should be provided, in case we run this migration and we have to undo the change, we won't be able to do it without the down behavior.